src/main/main.ts

```diff
@@ -39,14 +46,6 @@ let argv: string[] = [];
  */
 export async function onReady() {
   await onFirstRunMaybe();
-  if (!isDevMode()) {
-    process.env.NODE_ENV = 'production';
```

Comment:
We don't reference `NODE_ENV` anywhere else in the code so I figured it was safe to remove.